### PR TITLE
[SB-696] [SB-536] Updated the key for WHITELABEL

### PR DESCRIPTION
**Pull request by lkkadiri**
[SB-696](https://jira.sprucelabs.ai/jira/browse/SB-696)
[SB-536](https://jira.sprucelabs.ai/jira/browse/SB-536)

@sprucelabsai/engineers

## Description 
Updated the WHITELABEL env var name
## Type
- [ ] Feature
- [x] Bug
- [ ] Tech debt

## Steps to Test or Reproduce
Updated the WHITELABEL env var name

### CODE REVIEW

File: src/skillskit/next/_document.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default class MyDocument extends Document {
 		const sheet = new ServerStyleSheet()
 		const { auth, config } = store && store.getState()
 
-		let whitelabel = config.WHITELABEL_STYLESHEET_URL
+		let whitelabel = config.WHITELABEL
```
Comment (ryanrd) [inline src/skillskit/next/_document.js]: I had some questions re: original story, I'm going to mark this as do_not_merge until we can decide if this is the behavior we want

Comment (ryanrd) [reply src/skillskit/next/_document.js]: Is the skill configuration WHITELABEL supposed to:  override the Org wide whitelabel, be included in addition to, or be overridden by?  In this code Org whitelabel stylesheet would replace skill whitelabel stylesheet but I’m not sure if this is the behavior we want.  @liquidg3 / @artisgarbage can you weigh in?

**Comment by mcdermed** [comment]
This is working great locally with the changes https://github.com/sprucelabsai/sprucebot-skill-guest-feedback/pull/31

File: src/skillskit/next/_document.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default class MyDocument extends Document {
 
 	render() {
 		return (
-			<html className="skill">
+			<html className={`skill skill-${this.props.config.SLUG}`}>
```
Comment (liquidg3) [inline src/skillskit/next/_document.js]: assume config may not be set (when not authed)

Comment (mcdermed) [reply src/skillskit/next/_document.js]: Config is the result of our `pluck()` in `config`. It currently doesn't include properties from the server as far as I can tell

**Comment by lkkadiri** [comment]
@liquidg3 @ryanrd 
Added a check to see if config has SLUG and assigning a className if it does and also added the orgWhitelabel stylesheet to override all the others if it exists.

**Pull request merged**